server.js

```diff
@@ -35,7 +35,24 @@ function deploy(config, ready) {
         return path.posix.join(...url.split(path.sep));
     }
 
+    let logger = fs.createWriteStream('log.txt', {
+        flags: 'a'
+    });
+    let writeLine = (line) => logger.write(`\n${line}`);
+
     server.on('request', (request, response) => {
+        let now = new Date().toLocaleTimeString('en-US', {  hour: "2-digit", minute: "2-digit", hour12: false })
```

Comment:
I would prefer defining the format beforehand (in the `deploy` function) using
```javascript
const formatTime = Intl.DateTimeFormat('en-US', { hour: "2-digit", minute: "2-digit", second:"2-digit", hour12: false }).format
```

And using the `formatTime(Date.now())` in the `request` function